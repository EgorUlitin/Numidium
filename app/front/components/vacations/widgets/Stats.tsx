import * as React from 'react'
import * as moment from 'moment'
import { Card } from 'antd'

import Vacation from 'model/Vacation'

import Container from './StatsContainer'


export interface Props {
    vacations: Vacation[]
}

class Stats extends React.PureComponent<Props, {}> {

    render() {
        const { vacations } = this.props

        return (
            <Card title={'Статистика'}>
                <p>...</p>
            </Card>
        )
    }
}

export default Container(Stats)
