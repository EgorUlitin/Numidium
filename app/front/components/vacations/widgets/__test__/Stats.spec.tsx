import * as React from 'react'
import * as renderer from 'react-test-renderer'

import { Stats as StatsComponent } from '../Stats'

test('`Stats` widget renders correctly', () => {
    const tree = renderer
        .create(<StatsComponent vacations={[]} />)
        .toJSON()

    expect(tree).toMatchSnapshot()
})
