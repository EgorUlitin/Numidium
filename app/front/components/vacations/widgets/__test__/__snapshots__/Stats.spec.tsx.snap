// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Stats\` widget renders correctly 1`] = `
<div
  className="ant-card ant-card-bordered"
>
  <div
    className="ant-card-head"
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        Статистика
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={undefined}
  >
    <p>
      Coming soon...
    </p>
  </div>
</div>
`;
